pkg/apis/sippyprocessing/v1/types.go

```diff
@@ -79,6 +91,10 @@ type TopLevelIndicators struct {
 	// Upgrade indicates how successful we are with upgrading onto clusters that have already installed.
 	// Low Upgrade pass rates means clusters are not able to upgrade.  This should stop us from shipping the product.
 	Upgrade FailingTestResult
+	// Tests indicates how successful we are in running the e2e test suites.
+	Tests FailingTestResult
```

Comment:
hm.  This is based on how often the e2e container exits successfully?

presumably only counting job runs where the e2e container was started (i.e. throwing away runs where install failed or something else went wrong early)?

Reply:
I think the utility for TRT is that if we're staying on top of things - filing bugs, and adding exceptions to origin - then we should be holding this percentage steady or improving it.  I think there's also a useful comparison across releases - 4.8 is 70%+ (same for 4.7), but 4.9 is only 55% (but it is higher than last week).

Reply:
yeah, i just wonder about how noisy it will be since it's an aggregation across so many factors.  but sure, we can give it a shot.

Reply:
Ok, happy to remove it if it turns out not to be useful.  I added more to the tooltip, let me know if that isn't clear enough